var/spack/repos/builtin/packages/NCEPLIBS-ip/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class NceplibsIp(CMakePackage):
+    """The NCEP general interpolation library (iplib) contains Fortran
+    90 subprograms to be used for interpolating between nearly all
+    (rectilinear) grids used at NCEP."""
+
+    homepage = "https://github.com/NOAA-EMC/NCEPLIBS-ip"
+    git      = "git@github.com:NOAA-EMC/NCEPLIBS-ip"
```

Comment:
I don't think it's good to use ssh because if you don't have ssh keys setup on your machine it won't work.

If you do `spack create https://github.com/NOAA-EMC/NCEPLIBS-ip/archive/refs/tags/v3.3.3.tar.gz` with the release tar it will automatically fill in the version and checksum.

Reply:
Agree with Kyle's opinion

Reply:
I'm not sure how to change the package file to use the tarball...

Reply:
I would delete the package and run `spack create` again.

Reply:
OK, never mind, I get it...

Stand by for updates...